src/eddymotion/estimator.py

```diff
@@ -75,7 +75,8 @@ def fit(
             Number of parallel jobs.
         seed : :obj:`int` or :obj:`bool`
             Seed the random number generator (necessary when we want deterministic
-            estimation).
+            estimation). If an integer, the value is used to initialize the
+            generator; if True, an internal value is used to initialize it.
```

Comment:
```suggestion
            generator; if ``True`` or ``0``, the arbitrary value of ``20210324`` is used to initialize it.
```

Reply:
Done in b0e53e4. Generally, I do not think adding hard-coded values to the docstring is a good practice: the risk of changing the value for some reason and forgetting to update the docstring is usually high. Best would be to declare the value in some utils file, use the value throughout the code, and cross-reference it in the docstrings.

For the time being, I have changed the docstring as suggested though.

Reply:
That's a good point. This value should be a static variable of the module with its own docstring.